src/main/java/org/testcontainers/containers/TarantoolContainerImageHelper.java

```diff
@@ -71,6 +72,7 @@ private static boolean hasImage(String tag) {
         final List<Image> images = getDockerClient().listImagesCmd().exec();
         return images.stream()
                 .map(Image::getRepoTags)
+                .filter(Objects::nonNull)
```

Comment:
Should we move this below right before `.anyMatch`?
I thought that exactly repoTag was null

Reply:
No, we can't. In this case array of tags is null.

Reply:
I believe you that you have tested and got success result